messages/pt.json

```diff
@@ -4,6 +4,9 @@
   "store/shipping-calculator.edit": "alterar",
   "store/shipping-calculator.estimate": "calcular",
   "store/shipping-calculator.viewDeliveryOptions": "Ver opções de entrega",
+  "store/shipping-calculator.allItemsUnavailable": "Nenhum dos produtos pode ser entregue nesse endereço.",
+  "store/shipping-calculator.unavailableItems": "Atenção a {numberOfUnavailableItems, plural, one {# produto que não pode ser entregue} other {# produtos que não podem ser entregues}} neste endereço.",
+  "store/shipping-calculator.optionsForRemainingProducts": "Opções para os produtos restantes",
```

Comment:
"Atenção a um produto que não pode ser entregue neste endereço" is a bit different from the `en` message. Could you confirm with @davicosta99 what would be the message in Portuguese?

Reply:
It was @augustocb who told me the `pt-br` translation.

Reply:
We just discussed this but we couldn't came to a conclusion. We are waiting for @davicosta99 to say what he thinks.